<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mayur" failures="1" tests="6" name="com.Remosys.Wevend.TestLayer.TestGumBallPage" time="3.309" errors="0" timestamp="2023-11-20T11:26:39 IST" skipped="4">
  <testcase classname="com.Remosys.Wevend.TestLayer.TestGumBallPage" name="testGumballLaunch" time="1.087"/>
  <system-out/>
  <testcase classname="com.Remosys.Wevend.TestLayer.TestGumBallPage" name="testSuccessfullPayment" time="0.190">
    <skipped/>
  </testcase> <!-- testSuccessfullPayment -->
  <system-out/>
  <testcase classname="com.Remosys.Wevend.TestLayer.TestGumBallPage" name="testNavigationToPaymentPage" time="0.000">
    <skipped/>
  </testcase> <!-- testNavigationToPaymentPage -->
  <system-out/>
  <testcase classname="com.Remosys.Wevend.TestLayer.TestGumBallPage" name="testNavigationToCheckoutPage" time="0.000">
    <skipped/>
  </testcase> <!-- testNavigationToCheckoutPage -->
  <system-out/>
  <testcase classname="com.Remosys.Wevend.TestLayer.TestGumBallPage" name="testPaymentProcess" time="0.000">
    <skipped/>
  </testcase> <!-- testPaymentProcess -->
  <system-out/>
  <testcase classname="com.Remosys.Wevend.TestLayer.TestGumBallPage" name="testAddToCartFunctionality" time="2.032">
    <failure message="The following asserts failed:
	expected [2] but found [1]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2] but found [1]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.Remosys.Wevend.TestLayer.TestGumBallPage.testAddToCartFunctionality(TestGumBallPage.java:99)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testAddToCartFunctionality -->
  <system-out/>
</testsuite> <!-- com.Remosys.Wevend.TestLayer.TestGumBallPage -->
